assets/css/style.css

```diff
@@ -301,12 +332,21 @@ footer {
 /**************
  * Shabad Page
  **************/
+blockquote {
```

Comment:
Are blockquotes used outside of .search-results? Why are there two separate rules?